src/Microsoft.Kusto.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -446,11 +446,9 @@ private async Task<ConnectionCompleteParams> TryOpenConnection(ConnectionInfo co
             try
             {
                 connectionInfo.ConnectionDetails.Pooling = false;
-                // build the connection string from the input parameters
-                string connectionString = BuildConnectionString(connectionInfo.ConnectionDetails);
 
-                // create a sql connection instance
-                connection = connectionInfo.Factory.CreateDataSourceConnection(connectionString, connectionInfo.ConnectionDetails.AzureAccountToken, connectionInfo.OwnerUri);
+                // create a kusto connection instance
+                connection = connectionInfo.Factory.CreateDataSourceConnection(connectionInfo.ConnectionDetails, connectionInfo.OwnerUri);
```

Comment:
Changed logic to pass around ConnectionDetails instead of connectionString

Reply:
This comment should be "create a data source connection instance". There is nothing specific to Kusto here. Or maybe remove the comment as the function name is self-explanatory.